src/app/modules/item/pages/group-progress-grid/group-progress-grid.component.ts

```diff
@@ -138,12 +141,22 @@ export class GroupProgressGridComponent implements OnChanges, OnDestroy {
   }
 
   showProgressDetail(target: HTMLElement, userProgress: TeamUserProgress, row: DataRow, col: DataColumn): void {
+    if (!this.itemData) {
+      throw new Error('Unexpected: Missed item data');
+    }
     if (!this.group) {
       throw new Error('Unexpected: Missed group');
     }
     this.progressOverlay = {
       target,
       progress: userProgress,
+      ...(this.currentFilter !== 'Groups' && col.type === 'Task' ? {
+        taskDetails: {
+          route: rawItemRoute(typeCategoryOfItem(col), col.id),
```

Comment:
You have all the info to build the full route with the path. It will be easier to create the full route here and to just add the `answerId` in the template with the specific pipe.

Reply:
Fixed